tests/unit/components/cylc/analysis/analysistimeseries.vue.spec.js

```diff
@@ -24,20 +24,16 @@ import TimeSeries from '@/components/cylc/analysis/TimeSeries.vue'
 
 const vuetify = createVuetify()
 const analysisJobs = analysisJobQuery.data.jobs
-const analysisTasks = ['eventually_succeeded', 'succeeded', 'waiting']
 const $workflowService = sinon.createStubInstance(WorkflowService)
 
 describe('TimeSeries component', () => {
-  /**
-   * @param options
-   * @returns {Wrapper<Tree>}
-   */
-  const mountFunction = options => {
+  const mountFunction = (options) => {
     return mount(TimeSeries, {
       global: {
         plugins: [vuetify],
         mocks: { $workflowService },
       },
+      shallow: true,
```

Comment:
Shallow mount to avoid
```
[Vue warn]: Component is missing template or render function. 
  at <VueApexCharts type="rangeBar" options= {
...
```

(Don't need to mount VueApexCharts for the purposes of these unit tests)